programs/helium-sub-daos/src/instructions/calculate_utility_score_v0.rs

```diff
@@ -87,16 +100,31 @@ pub fn handler(
     let info: Account<DaoEpochInfoV0> = Account::try_from(&ctx.accounts.prev_dao_epoch_info)?;
     prev_supply = info.current_hnt_supply;
     prev_total_utility_score = info.total_utility_score;
+    prev_not_emitted = info.num_not_emitted;
   }
 
+  if ctx.accounts.not_emitted_counter.lamports() > 0
+    && !ctx
+      .accounts
+      .not_emitted_counter
+      .to_account_info()
+      .data_is_empty()
+  {
+    let info: Account<NotEmittedCounterV0> = Account::try_from(&ctx.accounts.not_emitted_counter)?;
+    not_emitted = info.amount_not_emitted.saturating_sub(prev_not_emitted);
+  };
+
+  ctx.accounts.dao_epoch_info.num_not_emitted = not_emitted;
```

Comment:
Shouldn't this keep track of the `total` not emitted. Right now it's only keeping track of what was not emitted last epoch. I don't think this calculation will work with taking the info.amount_not_emitted and subbing prev_not_emitted. You'd need to have prev_not_emittebe the previous total.